tests/vm_test.go

```diff
@@ -1292,7 +1292,7 @@ func waitForVMStateTransition(vm *v1.VirtualMachine, expectedStates []v1.Virtual
 func powerOff(vmi *v1.VirtualMachineInstance) {
 	// Can't use SafeExpectBatch since we may not get a prompt after the command
 	ExpectWithOffset(1, console.ExpectBatch(vmi, []expect.Batcher{
-		&expect.BSnd{S: "sudo poweroff\n"},
-		&expect.BExp{R: "The system is going down NOW!"},
+		&expect.BSnd{S: "poweroff\n"},
+		&expect.BExp{R: console.PromptExpression},
```

Comment:
Getting a prompt back after a call to `poweroff` is not guaranteed, this bit us before. If the `poweroff` command itself on alpine doesn't produce any output, then maybe we shouldn't expect anything.